openfisca_dummy_country/model/model.py

```diff
@@ -35,10 +35,10 @@ class birth(Variable):
     definition_period = ETERNITY
```

Comment:
> This comment concerns the whole `openfisca_dummy_country.model.model` Python module: consider moving the file to `openfisca_dummy_country/model.py` otherwise callers will have to import `openfisca_dummy_country.model.model` which is strange :)